### PR TITLE
[11.0][MIG] marketing_crm_partner: Migration to 11.0

### PR DESCRIPTION
**Pull request by cristinamartinrod**
This module extends the functionality of the CRM to support having the tracking fields available in the partner and copy them there automatically when the partner is created from a lead/opportunity.

cc @Tecnativa

### CODE REVIEW

**Comment by cristinamartinrod** [comment]
**Fix Test - WIP**

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,4 @@ docs/_build/
 # Backup files
 *~
 *.swp
+marketing_crm_partner/.DS_Store
```
Comment (simahawk) [inline .gitignore]: this should be *.DS_Store

**Review by simahawk — APPROVED** [review]
LG. minor issue w/ .gitignore

**Comment by simahawk** [comment]
tests do not pass.

**Comment by simahawk** [comment]
@cristinamartinrod pls add [WIP] to title :)

**Review by rafaelbn — APPROVED** [review]

File: marketing_crm_partner/models/crm_lead.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class CRMLead(models.Model):
+    _inherit = "crm.lead"
+
+    @api.model
+    def _create_lead_partner_data(self, name, is_company, parent_id=False):
+        """Populate marketing fields in partner."""
+        res = super()._create_lead_partner_data(
+            name, is_company, parent_id=parent_id,
+        )
+        for _key, field, _cookie in self.tracking_fields():
```
Comment (simahawk) [inline marketing_crm_partner/models/crm_lead.py]: @pedrobaeza on my 1st review I was about to ask for this change (use self.tracking_fields) but then I read the code in utm.mixin where it's clearly stated that we are not supposed to use the one on self but the one on the mixin straight (as it was before). 

Check [this](https://github.com/odoo/odoo/blob/11.0/addons/utm/models/utm.py#L47).
If you have a better insight... I'd love to hear that! :wink:

File: .gitignore (hunk 1 above)
Comment (cristinamartinrod) [reply .gitignore]: Done!

File: marketing_crm_partner/models/crm_lead.py (hunk 2 above)
Comment (pedrobaeza) [reply marketing_crm_partner/models/crm_lead.py]: You are right, Simone! I switch back to `self.env['utm.mixin'].tracking_fields()` for not preventing overridings over the mixin.

**Comment by pedrobaeza** [comment]
Merging this

**Pull request merged**